react/features/base/conference/actions.js

```diff
@@ -437,13 +437,15 @@ export function createConference() {
         const config = state['features/base/config'];
         const { tenant } = state['features/base/jwt'];
         const { email, name: nick } = getLocalParticipant(state);
+        const customDomain = /_[-\da-f]{36}$/.test(room) ? `breakout.${config.hosts.domain}` : null;
```

Comment:
I have moved this logic to [createRoom in lib-jitsi-meet](https://github.com/jitsi/lib-jitsi-meet/pull/1654/files#diff-d86e76d490765474507e19efc80ad0127d74bad1b3dc31a05a4880838fd2b311). This seems to be a better place to handle this detection, rather than in the client app using the customDomain workaround.